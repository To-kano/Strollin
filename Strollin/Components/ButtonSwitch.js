import React from 'react';
import { connect } from 'react-redux';
import { StyleSheet, Image, TouchableHighlight } from 'react-native';
import Icon from './components/Icon';

function ButtonSwitch({statue, iconOn, iconOff, onPressOn, onPressOff, position}) {

  if (statue) {
    return (
      <TouchableHighlight
        onPress={onPressOn}
        style={{
          backgroundColor: "#ffffff",
          position: "absolute",
          top: position.top,
          right: position.right,
          padding: 16,
          borderRadius: 32,
          shadowColor: "#000",
          shadowOffset: {
              width: 0,
              height: 5,
          },
          shadowOpacity: 0.34,
          shadowRadius: 6.27,

          elevation: 10,
      }}>
        <Icon name={iconOn} size={29} color='#000000'/>
      </TouchableHighlight>
    );
  } else {
    return (
      <TouchableHighlight
        onPress={onPressOff}
        style={{
          backgroundColor: "#ffffff",
          position: "absolute",
          top: position.top,
          right: position.right,
          padding: 16,
          borderRadius: 32,
          shadowColor: "#000",
          shadowOffset: {
              width: 0,
              height: 5,
          },
          shadowOpacity: 0.34,
          shadowRadius: 6.27,

          elevation: 10,
        }}>
          <Icon name={iconOff} size={29} color='#000000'/>
        </TouchableHighlight>
    );
  }
}

//const mapStateToProps = (state) => state;
//export default connect(mapStateToProps)(ButtonIcon);
export default ButtonSwitch;
