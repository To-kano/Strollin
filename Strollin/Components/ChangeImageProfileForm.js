import React, { useState } from 'react';
import { launchImageLibrary } from 'react-native-image-picker';
import { StyleSheet, Image, Button, View, Text, TouchableOpacity } from 'react-native';
import { connect } from 'react-redux';
import { uploadImageProfile } from '../apiServer/image';
import ImageProfile from './components/ImageProfile';

const globalStyles = require('../Styles');

function ChangeImageProfileForm({ profil, dispatch, modalVisible, setModalVisible }) {

  const [image, setImage] = useState(null);

  const handleChooseImage = () => {
    launchImageLibrary({ noData: true }, (response) => {
      if (response) {
        setImage(response);
      }
    });
  };

  return (
    <>
      {image && (
        <View style={{ alignItems: 'center', width: '100%' }}>
          <Image
            source={{ uri: image.uri }}
            style={{ width: 128, height: 128, borderRadius: 16, marginBottom: 32 }}
          />
          <View style={{
            width: '100%',
            flexDirection: 'row',
            alignItems: 'center',
            justifyContent: 'space-between'
          }}>
            <TouchableOpacity 
              style={{ backgroundColor: "#ffffff", padding: 16, alignItems: 'center', borderRadius: 32 }}
              onPress={() => { setImage(null) }}
            >
              <Text style={globalStyles.paragraphs}>Annuler</Text>
            </TouchableOpacity>
            <TouchableOpacity
              style={{ backgroundColor: "#0989FF", padding: 16, alignItems: 'center', borderRadius: 32 }}
              onPress={() => {
                if (image) {
                  uploadImageProfile(profil.access_token, image).then((answer) => {
                    if (answer.image) {
                      const action = { type: 'SET_IMAGE_PROFILE', value: answer.image?.id };
                      dispatch(action);
                    }
                  });
                }
              }}
            >
              <Text style={[globalStyles.paragraphs, { color: '#ffffff' }]}>Confirmer</Text>
            </TouchableOpacity>
          </View>
        </View>
      )}

      {!image && (
        <View style={{ alignItems: 'center', width: '100%' }} >
          <ImageProfile style={{ width: 128, height: 128, borderRadius: 16, marginBottom: 32 }} />
          <View style={{
            width: '100%',
            flexDirection: 'row',
            alignItems: 'center',
            justifyContent: 'space-between'
          }}>
            <TouchableOpacity style={{ backgroundColor: "#ffffff", padding: 16, alignItems: 'center', borderRadius: 32 }} onPress={() => setModalVisible(!modalVisible)}>
              <Text style={globalStyles.paragraphs}>Annuler</Text>
            </TouchableOpacity>
            <TouchableOpacity style={{ backgroundColor: "#0989FF", padding: 16, alignItems: 'center', borderRadius: 32 }} onPress={handleChooseImage}>
              <Text style={[globalStyles.paragraphs, { color: '#ffffff' }]}>Choisir une image</Text>
            </TouchableOpacity>
          </View>
        </View>
      )}
    </>
  )
}

const styles = StyleSheet.create({
  icon: {
    width: 35,
    height: 35,
    resizeMode: 'contain',
    backgroundColor: 'white',
    borderColor: 'gray'
  },
});

const mapStateToProps = (state) => state;
export default connect(mapStateToProps)(ChangeImageProfileForm);
//export default ChangeImageProfileForm;