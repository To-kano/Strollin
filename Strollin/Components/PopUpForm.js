import React, { Component } from 'react';
import {
  Text, View, Button
} from 'react-native';

class PopUpForm extends Component {
  render() {
    return (
      <View style={{
        height: this.props.height, width: this.props.width, backgroundColor: '#FDFEFE', justifyContent: 'center'
      }}
      >
        <Text style={{
          textAlign: 'center', fontSize: 35, fontFamily: 'lobster', color: 'blue'
        }}
        >
          do you want to go to
          "
          {this.props.place}
          " ?
        </Text>
        <Button
          title="Yes"
          onPress={() => {
          }}
        />
        <Button
          title="No"
          onPress={() => {
          }}
        />

      </View>
    );
  }
}

export { PopUpForm };
