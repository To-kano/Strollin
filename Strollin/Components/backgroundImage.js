
import React, {useState} from 'react';
import { Image} from 'react-native';

function RandPic() {
    var nb = Math.floor(Math.random() * 3) + 1;
    var img = '';
  
    if (nb == 1)
      img = require('../ressources/street1.jpg');
    if (nb == 2)
      img = require('../ressources/street2.jpeg');
    if (nb == 3)
      img = require('../ressources/plum2.jpg');
    //console.log(nb)
    return img;
}

function BackgroundImage(props) {

  const [Img, onChangeImg] = useState(RandPic());

  return (
    <Image style={{width: '100%', height: '100%', resizeMode: 'cover', position: 'absolute'}}
    source={Img} />
  );

}

export default BackgroundImage;