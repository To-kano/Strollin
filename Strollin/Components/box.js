import {
  Button, Image, View, StyleSheet, Text, ScrollView, FlatList
} from 'react-native';
import React from 'react';

async function GotoComment(props) {
  //console.log('GOING TO COMMENT');
  // redux currentComment = props.data.id
  props.navigation.navigate('CommentScreen');
}

function Box(props) {
  // console.log("props = ", props);
  return (
    <View style={{
      justifyContent: 'space-around', flex: 1, marginTop: 20, marginHorizontal: '10%', backgroundColor: 'rgba(255,255,255, 0.9)', borderRadius: 20, width: '80%'
    }}
    >
      <Text style={[{
        textAlign: 'center', fontSize: 30, color: '#39A5D6', margin: 5
      }]}
      >
        {props.name}
      </Text>
      <Text style={[{ fontSize: 25, marginLeft: '5%' }]}>
        Budget :
        {props.budget}
      </Text>
      <Text style={[{ fontSize: 25, marginLeft: '5%' }]}>
        PÃ©riode :
        {props.period}
      </Text>
      <Text style={[{ fontSize: 25, marginLeft: '5%' }]}>
        1/
        {props.destinations[0]}
      </Text>
      <Text style={[{ fontSize: 25, marginLeft: '5%' }]}>
        2/
        {props.destinations[1]}
      </Text>
      <Text style={[{ fontSize: 25, marginLeft: '5%' }]}>
        3/
        {props.destinations[2]}
      </Text>
      <Button
        title="Commentaires"
        onPress={() => GotoComment(props)}
      />
    </View>
  );
}

/* import I18n from "../Translation/configureTrans";

export default class Box extends Component {
  render() {
    return (
      <View style={styles.cont}>
        <Text style={{ fontSize: 40 }}> {I18n.t("trendingTrip")} </Text>
        <View style={styles.cont}>
          <Image style={{ resizeMode: 'stretch' }} source={require('../ressources/plum2.jpg')} />
        </View>
      </View>
    );
  }
} */

export default Box;

const styles = StyleSheet.create({
  cont: {
    flexDirection: 'column',
    justifyContent: 'flex-start',
    marginBottom: 20,
  },
  text: {
    fontSize: 22,
    marginBottom: 5,
    opacity: 0.5
  },
  img: {
    flexDirection: 'column',
    justifyContent: 'flex-start',
    alignItems: 'center',
    flex: 0.1,
    backgroundColor: 'red',
    width: '80%'
  },
  whiteBox: {
    justifyContent: 'space-between',
    flex: 1,
    marginTop: 20,
    marginHorizontal: '10%',
    backgroundColor: 'rgba(255,255,255, 0.9)',
    borderRadius: 20,
    textAlign: 'left',
    width: '80%',
    resizeMode: 'stretch',
  }
});
