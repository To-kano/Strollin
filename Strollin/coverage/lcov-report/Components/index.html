
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for Components</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../index.html">All files</a> Components</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">17.33% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>74/427</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">3.33% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>3/90</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">9.34% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>17/182</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">18.16% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>73/402</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line low'></div>
    <div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file high" data-value="ButtonIcon.js"><a href="ButtonIcon.js.html">ButtonIcon.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	</tr>

<tr>
	<td class="file high" data-value="Comment.js"><a href="Comment.js.html">Comment.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	</tr>

<tr>
	<td class="file high" data-value="CommentScreen.js"><a href="CommentScreen.js.html">CommentScreen.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="5" class="abs high">5/5</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="5" class="abs high">5/5</td>
	</tr>

<tr>
	<td class="file low" data-value="CourseEvaluation.js"><a href="CourseEvaluation.js.html">CourseEvaluation.js</a></td>
	<td data-value="35.29" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 35%"></div><div class="cover-empty" style="width: 65%"></div></div>
	</td>
	<td data-value="35.29" class="pct low">35.29%</td>
	<td data-value="17" class="abs low">6/17</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="22.22" class="pct low">22.22%</td>
	<td data-value="9" class="abs low">2/9</td>
	<td data-value="31.25" class="pct low">31.25%</td>
	<td data-value="16" class="abs low">5/16</td>
	</tr>

<tr>
	<td class="file medium" data-value="Form.js"><a href="Form.js.html">Form.js</a></td>
	<td data-value="50" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 50%"></div><div class="cover-empty" style="width: 50%"></div></div>
	</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	</tr>

<tr>
	<td class="file low" data-value="FriendList.js"><a href="FriendList.js.html">FriendList.js</a></td>
	<td data-value="25" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 25%"></div><div class="cover-empty" style="width: 75%"></div></div>
	</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="36" class="abs low">9/36</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="6" class="abs low">2/6</td>
	<td data-value="5.88" class="pct low">5.88%</td>
	<td data-value="17" class="abs low">1/17</td>
	<td data-value="28.13" class="pct low">28.13%</td>
	<td data-value="32" class="abs low">9/32</td>
	</tr>

<tr>
	<td class="file low" data-value="HistoryElement.js"><a href="HistoryElement.js.html">HistoryElement.js</a></td>
	<td data-value="36.36" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 36%"></div><div class="cover-empty" style="width: 64%"></div></div>
	</td>
	<td data-value="36.36" class="pct low">36.36%</td>
	<td data-value="11" class="abs low">4/11</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="16.67" class="pct low">16.67%</td>
	<td data-value="6" class="abs low">1/6</td>
	<td data-value="36.36" class="pct low">36.36%</td>
	<td data-value="11" class="abs low">4/11</td>
	</tr>

<tr>
	<td class="file low" data-value="HistoryNav.js"><a href="HistoryNav.js.html">HistoryNav.js</a></td>
	<td data-value="14.29" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 14%"></div><div class="cover-empty" style="width: 86%"></div></div>
	</td>
	<td data-value="14.29" class="pct low">14.29%</td>
	<td data-value="14" class="abs low">2/14</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="10" class="abs low">0/10</td>
	<td data-value="15.38" class="pct low">15.38%</td>
	<td data-value="13" class="abs low">2/13</td>
	</tr>

<tr>
	<td class="file low" data-value="Home.js"><a href="Home.js.html">Home.js</a></td>
	<td data-value="25" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 25%"></div><div class="cover-empty" style="width: 75%"></div></div>
	</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="4" class="abs low">1/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="4" class="abs low">1/4</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="4" class="abs low">1/4</td>
	</tr>

<tr>
	<td class="file low" data-value="HomePage.js"><a href="HomePage.js.html">HomePage.js</a></td>
	<td data-value="12.12" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 12%"></div><div class="cover-empty" style="width: 88%"></div></div>
	</td>
	<td data-value="12.12" class="pct low">12.12%</td>
	<td data-value="33" class="abs low">4/33</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="14" class="abs low">0/14</td>
	<td data-value="12.5" class="pct low">12.5%</td>
	<td data-value="32" class="abs low">4/32</td>
	</tr>

<tr>
	<td class="file low" data-value="LoginPage.js"><a href="LoginPage.js.html">LoginPage.js</a></td>
	<td data-value="17.95" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 17%"></div><div class="cover-empty" style="width: 83%"></div></div>
	</td>
	<td data-value="17.95" class="pct low">17.95%</td>
	<td data-value="39" class="abs low">7/39</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="14" class="abs low">0/14</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="13" class="abs low">0/13</td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="35" class="abs low">7/35</td>
	</tr>

<tr>
	<td class="file low" data-value="Notation.js"><a href="Notation.js.html">Notation.js</a></td>
	<td data-value="33.33" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 33%"></div><div class="cover-empty" style="width: 67%"></div></div>
	</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="6" class="abs low">2/6</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="5" class="abs low">1/5</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="6" class="abs low">2/6</td>
	</tr>

<tr>
	<td class="file high" data-value="PopUpForm.js"><a href="PopUpForm.js.html">PopUpForm.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="3" class="abs low">1/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	</tr>

<tr>
	<td class="file low" data-value="ProfileScreen.js"><a href="ProfileScreen.js.html">ProfileScreen.js</a></td>
	<td data-value="13.64" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 13%"></div><div class="cover-empty" style="width: 87%"></div></div>
	</td>
	<td data-value="13.64" class="pct low">13.64%</td>
	<td data-value="22" class="abs low">3/22</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="12" class="abs low">0/12</td>
	<td data-value="15" class="pct low">15%</td>
	<td data-value="20" class="abs low">3/20</td>
	</tr>

<tr>
	<td class="file low" data-value="Socket.js"><a href="Socket.js.html">Socket.js</a></td>
	<td data-value="10.2" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 10%"></div><div class="cover-empty" style="width: 90%"></div></div>
	</td>
	<td data-value="10.2" class="pct low">10.2%</td>
	<td data-value="49" class="abs low">5/49</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="10" class="abs low">0/10</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="11" class="abs low">0/11</td>
	<td data-value="10.64" class="pct low">10.64%</td>
	<td data-value="47" class="abs low">5/47</td>
	</tr>

<tr>
	<td class="file low" data-value="TagSelection.js"><a href="TagSelection.js.html">TagSelection.js</a></td>
	<td data-value="25" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 25%"></div><div class="cover-empty" style="width: 75%"></div></div>
	</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="12" class="abs low">3/12</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="7" class="abs low">0/7</td>
	<td data-value="27.27" class="pct low">27.27%</td>
	<td data-value="11" class="abs low">3/11</td>
	</tr>

<tr>
	<td class="file medium" data-value="TendanceSearchBar.js"><a href="TendanceSearchBar.js.html">TendanceSearchBar.js</a></td>
	<td data-value="50" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 50%"></div><div class="cover-empty" style="width: 50%"></div></div>
	</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="6" class="abs medium">3/6</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="3" class="abs low">1/3</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="6" class="abs medium">3/6</td>
	</tr>

<tr>
	<td class="file low" data-value="TripNavigation.js"><a href="TripNavigation.js.html">TripNavigation.js</a></td>
	<td data-value="6.06" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 6%"></div><div class="cover-empty" style="width: 94%"></div></div>
	</td>
	<td data-value="6.06" class="pct low">6.06%</td>
	<td data-value="33" class="abs low">2/33</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="14" class="abs low">0/14</td>
	<td data-value="6.45" class="pct low">6.45%</td>
	<td data-value="31" class="abs low">2/31</td>
	</tr>

<tr>
	<td class="file low" data-value="TripSuggestion.js"><a href="TripSuggestion.js.html">TripSuggestion.js</a></td>
	<td data-value="6.9" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 6%"></div><div class="cover-empty" style="width: 94%"></div></div>
	</td>
	<td data-value="6.9" class="pct low">6.9%</td>
	<td data-value="29" class="abs low">2/29</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="12" class="abs low">0/12</td>
	<td data-value="7.41" class="pct low">7.41%</td>
	<td data-value="27" class="abs low">2/27</td>
	</tr>

<tr>
	<td class="file low" data-value="UserRegister.js"><a href="UserRegister.js.html">UserRegister.js</a></td>
	<td data-value="9.09" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 9%"></div><div class="cover-empty" style="width: 91%"></div></div>
	</td>
	<td data-value="9.09" class="pct low">9.09%</td>
	<td data-value="33" class="abs low">3/33</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="12" class="abs low">0/12</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="13" class="abs low">0/13</td>
	<td data-value="9.38" class="pct low">9.38%</td>
	<td data-value="32" class="abs low">3/32</td>
	</tr>

<tr>
	<td class="file low" data-value="backgroundImage.js"><a href="backgroundImage.js.html">backgroundImage.js</a></td>
	<td data-value="0" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 0%"></div><div class="cover-empty" style="width: 100%"></div></div>
	</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="11" class="abs low">0/11</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="6" class="abs low">0/6</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="8" class="abs low">0/8</td>
	</tr>

<tr>
	<td class="file low" data-value="box.js"><a href="box.js.html">box.js</a></td>
	<td data-value="40" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 40%"></div><div class="cover-empty" style="width: 60%"></div></div>
	</td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="5" class="abs low">2/5</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="3" class="abs low">1/3</td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="5" class="abs low">2/5</td>
	</tr>

<tr>
	<td class="file low" data-value="map.js"><a href="map.js.html">map.js</a></td>
	<td data-value="1.96" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 1%"></div><div class="cover-empty" style="width: 99%"></div></div>
	</td>
	<td data-value="1.96" class="pct low">1.96%</td>
	<td data-value="51" class="abs low">1/51</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="28" class="abs low">0/28</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="15" class="abs low">0/15</td>
	<td data-value="2" class="pct low">2%</td>
	<td data-value="50" class="abs low">1/50</td>
	</tr>

<tr>
	<td class="file high" data-value="rondForm.js"><a href="rondForm.js.html">rondForm.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	</tr>

<tr>
	<td class="file high" data-value="sugest.js"><a href="sugest.js.html">sugest.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	</tr>

</tbody>
</table>
</div>
                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Sun Jan 03 2021 21:28:05 GMT+0100 (Central European Standard Time)
            </div>
        </div>
        <script src="../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../sorter.js"></script>
        <script src="../block-navigation.js"></script>
    </body>
</html>
    