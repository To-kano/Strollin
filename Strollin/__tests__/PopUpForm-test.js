import React from 'react';
import {shallow} from 'enzyme';
import {PopUpForm} from '../Components/PopUpForm';
import renderer from 'react-test-renderer';


describe('PopUpForm', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<PopUpForm />)
            expect(component).toMatchSnapshot()
        });

        test('test render PopupForm', () => {
            renderer.create(
                <PopUpForm />,
            )      
          })
    });
});