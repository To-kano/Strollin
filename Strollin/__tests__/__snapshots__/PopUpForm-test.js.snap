// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopUpForm Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
